### ISSUE TITLE
 undefined is not an object (evaluating 'g.i18n[g.lang].dayOfWeek')

### ISSUE THREAD

**Issue by gobijan** [dszymczuk/dsEventCalendar#25]
Hi, 

I get this error when I try to create a new event (latest c5 5.7.X). Any clue what's wrong?

<img width="730" alt="bildschirmfoto 2016-04-28 um 18 24 06" src="https://cloud.githubusercontent.com/assets/49339/14893138/82ebd4d6-0d6e-11e6-8c69-e9302685f714.png">

**Comment by tigerxy** [comment]
Hey @GoBijan 
on which page you try to create a new event? Which browser? Pro Version?
Here is the wrong repo. Please open an issue [there](https://github.com/dszymczuk/dsEventCalendar57)

**Comment by dszymczuk** [comment]
Yep, please describe more info on [here](https://github.com/dszymczuk/dsEventCalendar57).

Or describe on Concrete5 support for this addon:
[dsEventCalendar 5.6 Free](https://www.concrete5.org/marketplace/addons/ds-event-calendar/)
[dsEventCalendar 5.6 Pro](https://www.concrete5.org/marketplace/addons/ds-event-calendar-pro/)
[dsEventCalendar 5.7 Free](https://www.concrete5.org/marketplace/addons/ds-event-calendar1/)
[dsEventCalendar 5.7 Pro](https://www.concrete5.org/marketplace/addons/ds-event-calendar-pro1/)
